server/cmwell-plugin-gremlin/src/main/scala/cmwell/plugins/impl/GremlinParser.scala

```diff
@@ -61,7 +59,12 @@ class GremlinParser extends SgEngineClient {
       typedPipe
     }
 
-    def read(p: Pipe[_, _]) = p.iterator().mkString("\n")
+    def read(p: Pipe[_, _]) = {
```

Comment:
Could that be done better?

Reply:
it works:
`p.iterator().asScala.mkString("\n")`